dist/redhat/build_rpm.sh

```diff
@@ -118,12 +116,4 @@ rpm_payload_opts=(--define "_binary_payload w2${xz_thread_param}.xzdio")
 
 ln -fv $RELOC_PKG $RPMBUILD/SOURCES/
 pystache dist/redhat/scylla.spec.mustache "{ \"version\": \"$SCYLLA_VERSION\", \"release\": \"$SCYLLA_RELEASE\", \"housekeeping\": $DIST, \"product\": \"$PRODUCT\", \"$PRODUCT\": true, \"reloc_pkg\": \"$RELOC_PKG_BASENAME\", $MUSTACHE_DIST }" > $RPMBUILD/SPECS/scylla.spec
-if [ "$TARGET" = "centos7" ]; then
-    rpmbuild -ba "${rpm_payload_opts[@]}" --define "_topdir $RPMBUILD" --define "dist .el7" $RPMBUILD/SPECS/scylla.spec
-elif [ "$TARGET" = "centos8" ]; then
-    rpmbuild -ba "${rpm_payload_opts[@]}" --define "_topdir $RPMBUILD" --define "dist .el8" $RPMBUILD/SPECS/scylla.spec
-elif [ "$TARGET" = "centos" ]; then
-    rpmbuild -ba "${rpm_payload_opts[@]}" --define "_topdir $RPMBUILD" $RPMBUILD/SPECS/scylla.spec
-else
-    rpmbuild -ba "${rpm_payload_opts[@]}" --define "_topdir $RPMBUILD" $RPMBUILD/SPECS/scylla.spec
-fi
+rpmbuild -ba "${rpm_payload_opts[@]}" --define "_topdir $RPMBUILD" $RPMBUILD/SPECS/scylla.spec
```

Comment:
Why did you add all those lines if you now remove them?

Reply:
It is an intermediate step (to have a general el instead of el7). I will squash everything when we are satisfied with the solution.